css/style.css

```diff
@@ -1,19 +1,73 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+.nav {
+  background-color: #fff;
+  position: fixed;
+  width: 100%;
+  border-bottom: 1px solid #ddd;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.nav .nav-links {
+  float: right;
+  color: #838994;
+  font-size: 1.04rem;
+  margin-top: 1.7rem;
+}
+.nav .nav-links li {
+  display: inline-block;
+  margin-left: 1.563rem;
+}
+.nav .e-logo {
+  height: 4.375rem;
+}
 
+.section-one {
+  background-image: url("https://www.ukyouth.org/wp-content/uploads/2020/01/finalhomepage-scaled.gif");
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 24.2rem;
+  overflow: auto;
+}
 
+.section-two .sec2 {
+  margin-top: 7.875rem;
+  padding: none;
+  color: #1d1e21;
+  text-align: center;
+}
+.section-two .buttons {
+  text-align: center;
+  margin-top: 1.563rem;
+}
+.lowe {
```

Comment:
Make sure you name things using easy-to-understand words. I'm not sure what `lowe` means.